open-company/open-company-storage
# NUX sample content

https://trello.com/c/z1G1SjZo

Supports https://github.com/open-company/open-company-web/pull/605

To test:


Use the `nux-2.0` web branch.

Go through the NUX, you should see:

- [x] List of 20 topics that matter in NUX that matches what's in the config here

Select at least these sections: All-hands, Decisions, Designs, Lessons learned, Marketing, People, Product updates, Week in review

Select a few additional ones as well.

- [x] Verify that there is a sample post in all these sections that matches the one in this repo: All-hands, Decisions, Designs, Lessons learned, Marketing, People, Product updates, Week in review
- [x] Verify there are not sample posts in any of the other sections you selected
- [x] Verify you have a General section (hard coded, not selected) and that it has a post that matches the one in this repo.

Now to look at draft sample...

- [x] Verify the NUX user sees Drafts (1) in the left nav. once logged in, and there is an about <name> draft post to finish w/ a NUX tip about it
- [x] Continue editing the post, verify the tip goes away
- [x] Post the post, verify it posts OK

Invite an admin and a contributor and a viewer to your new org. Log in as each.

- [x] Verify the author and admin invited user see Drafts (1) in the left nav. once logged in, and there is an about me draft post to finish w/ a NUX tip about it
- [x] Continue editing the post, verify the tip goes away
- [x] Post the post, verify it posts OK
- [x] Verify that the viewer doesn't see Drafts in the left nav.

Now to look at sample cleanup...

- [x] Create one more new post from scratch and post it
- [x] Pick one of the posted About posts, and click the delete option, do you just see the normal delete dialog? Just cancel it.
- [x] Pick any sample post, click the delete option, do you see the delete dialog w/ a delete all samples option? Click it.
- [x] Verify all the sample posts are gone
- [x] Verify none of the about posts are the new post from scratch are gone

Brilliant.

- [ ] Deploy with `nux-2.0` web when ready
